modules/veteran/app/sidekiq/rep_org_addresses/queue_address_updates.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sentry_logging'
+require 'roo'
+
+module RepOrgAddresses
+  # A Sidekiq job class for processing address and email updates from an Excel file.
```

Comment:
Just curious, do you know if there are automatic retries setup by default?

Reply:
Yes. Those cannot be disabled for Batches.